python_tests/build.rs

```diff
@@ -0,0 +1,32 @@
+use std::{env, path::Path, time::Instant};
+
+use rust_swig::{PythonConfig, LanguageConfig};
+
+fn main() {
+    env_logger::init();
+
+    let now = Instant::now();
+
+    let out_dir = env::var("OUT_DIR").unwrap();
+    rust_swig_expand(
+        Path::new("src/glue.rs.in"),
+        &Path::new(&out_dir).join("glue.rs"),
+    );
+    let expand_time = now.elapsed();
+    println!(
+        "rust swig expand time: {}",
+        expand_time.as_secs() as f64 + (expand_time.subsec_nanos() as f64) / 1_000_000_000.
+    );
+    println!("cargo:rerun-if-changed=src");
```

Comment:
According to cargo's book directory really is not supported:

https://doc.rust-lang.org/cargo/reference/build-scripts.html